recipe/meta.yaml

```
@@ -47,9 +47,10 @@ outputs:
     script: build-java.bat  # [win]
     build:
       error_overlinking: true
-      ignore_run_exports:
-        - libstdcxx-ng  # [linux]
-        - vs2015_runtime  # [win]
+      ignore_run_exports_from:
+        # we want to ignore libstdcxx, but not libgcc; the latter also comes in through
+        # compiler('c'), and conda keeps that despite ignoring the cxx run-exports
+        - {{ compiler('cxx') }}
```

@h-vetinari, can/should this be

```suggestion
        - {{ compiler('cxx') }}  # [not osx]
```

since the macOS builds seem to link against `libc++`?

---

Yeah, sorry, I had broken this while cleaning up something else.